warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/CompositeIngest.java

```diff
@@ -68,7 +68,9 @@ static boolean isOverloadedCompositeField(Multimap<String,String> compositeField
     
     static boolean isOverloadedCompositeField(Collection<String> compFields, String compositeFieldName) {
         if (compFields != null && !compFields.isEmpty())
-            return compFields.stream().findFirst().get().equals(compositeFieldName);
+            if (compFields.stream().findFirst().isPresent()) {
+                return compFields.stream().findFirst().get().equals(compositeFieldName);
+            }
```

Comment:
why not the following?

```suggestion
            return compositeFieldName.equals(compFields.stream().findFirst().orElse(null));
```